### PR TITLE
Community Choice 2017

### PR DESCRIPTION
**Pull request by Ephemeralis**
Place your bets for CBCC community choice 2017.

The contest links will be dead until I flip the switch. Review away.

### CODE REVIEW

File: news/2018-01-18-community-choice-2017.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+date: 2018-01-18 07:00:00 +0000
+---
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which maps were truly the greatest of the year. Have your say and help us figure out which maps from 2017 are truly the Community's Favourite!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+You may know them as the "best of" votes held historically throughout the past few years, but the votes are back for 2018 with a brand new look, and a better name!
+
+Each osu! player gets **10 votes per gamemode** to cast for their favourite ranked maps of 2017 that they've played. The voting is super simple - find your favourite 10 maps in any mode, shoot them a star, and you're done!
```
Comment (JBHyperion) [inline news/2018-01-18-community-choice-2017.md]: Is "osu!" necessary to be included here? Although it's clarified right after, it makes it sound like the voting is only for the osu! game here.

Comment (JBHyperion) [inline news/2018-01-18-community-choice-2017.md]: I assume you aren't required to use the full allocation if you don't want to? "decide on up to 10 maps in any mode" perhaps?

File: news/2018-01-18-community-choice-2017.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* [osu!](https://osu.ppy.sh/community/contests/59)
+* [osu!taiko](https://osu.ppy.sh/community/contests/60)
+* [osu!catch](https://osu.ppy.sh/community/contests/61)
+* [osu!mania](https://osu.ppy.sh/community/contests/62)
+
+The voting will run for **14 days** from the date of this post, so get cracking, and make sure to tell all your friends!
+
+We'll also be hosting a similar poll purely for Loved maps at a later date - this is why no Loved maps are included in the entries for this vote. Stay tuned!
+
+Which map do you think will win?
```
Comment (JBHyperion) [inline news/2018-01-18-community-choice-2017.md]: "maps" plural for covering all game modes? I personally also like a little reiteration of "The decision is yours!" or some similar empowering thing here, but maybe that's just me being corny.

File: news/2018-01-18-community-choice-2017.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+title: "Community Choice 2017"
+date: 2018-01-18 07:00:00 +0000
+---
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which maps were truly the greatest of the year. Have your say and help us figure out which maps from 2017 are truly the Community's Favourite!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+You may know them as the "best of" votes held historically throughout the past few years, but the votes are back for 2018 with a brand new look, and a better name!
```
Comment (TPGPL) [inline news/2018-01-18-community-choice-2017.md]: A few years

File: news/2018-01-18-community-choice-2017.md (hunk 1 above)
Comment (TPGPL) [reply news/2018-01-18-community-choice-2017.md]: Gamemode into game mode?

File: news/2018-01-18-community-choice-2017.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Each osu! player gets **10 votes per gamemode** to cast for their favourite ranked maps of 2017 that they've played. The voting is super simple - find your favourite 10 maps in any mode, shoot them a star, and you're done!
+
+The highest voted map for each gamemode will be declared the Community Favourite of 2017 and earn its mapper **a shiny new profile badge, and 6 months of osu!supporter** for their efforts.
```
Comment (TPGPL) [inline news/2018-01-18-community-choice-2017.md]: Game mode
Community's Favourite?

Comment (TPGPL) [reply news/2018-01-18-community-choice-2017.md]: Mapper into creator?

File: news/2018-01-18-community-choice-2017.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+The top 10 maps for each mode will also grant their mappers **1 month of osu!supporter** as well.
```
Comment (TPGPL) [inline news/2018-01-18-community-choice-2017.md]: Mappers into Creators

File: news/2018-01-18-community-choice-2017.md (hunk 2 above)
Comment (TPGPL) [reply news/2018-01-18-community-choice-2017.md]: Which beatmap

File: news/2018-01-18-community-choice-2017.md (hunk 4 above)
Comment (Ephemeralis) [reply news/2018-01-18-community-choice-2017.md]: community favourite is exalted singular so works proper in this instance

File: news/2018-01-18-community-choice-2017.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Community Choice 2017"
+date: 2018-01-18 07:00:00 +0000
+---
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which maps were truly the greatest of the year. Have your say and help us figure out which maps from 2017 are truly the Community's Favourite!
```
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2018-01-18-community-choice-2017.md]: "your say" is colloquial or casual for "your vote"

File: news/2018-01-18-community-choice-2017.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which beatmaps were truly the greatest of the year. Cast your votes and help us figure out which maps from 2017 are truly the Community Favourites!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+You may know them as the "best of" votes held historically throughout the past few years, but the votes are back for 2018 with a brand new look, and a better name!
+
+Everyone gets **10 votes per game mode** to cast for their favourite ranked maps of 2017 that they've played at least once before. The voting is super simple - find up to 10 of your favourite maps in any mode, shoot them a star via the contest listing, and you're done!
+
+The highest voted map for each gamemode will be declared the Community Favourite of 2017 and earn its mapper **a shiny new profile badge, and 6 months of osu!supporter** for their efforts.
```
Comment (Xinnoh) [inline news/2018-01-18-community-choice-2017.md]: Change to -> "the Favourite of 2017, earning its mapper a shiny new profile badge and 6 months of"
uses 'and' less, and comma no longer seperates related clauses

**Pull request merged**